services/port_manager/src/main/java/com/futurewei/alcor/portmanager/processor/AbstractProcessor.java

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.portmanager.processor;
+
+import org.slf4j.Logger;
```

Comment:
Can we use Alcor logger or you need something special from slf4j?

Reply:
I thought alcor would use slf4j directly.